packages/use-sync-external-store/index.js

```diff
@@ -9,4 +9,4 @@
 
 'use strict';
 
-export {useSyncExternalStore} from './src/useSyncExternalStore';
```

Comment:
Pretty sure these get re-written automatically and should stay relative imports: https://unpkg.com/browse/use-sync-external-store@1.2.0/index.js